src/power_grid_model/validation/rules.py

```diff
@@ -70,12 +70,12 @@
     ValidationError,
 )
 from power_grid_model.validation.utils import (
-    eval_expression,
-    get_indexer,
-    get_mask,
-    get_valid_ids,
-    nan_type,
-    set_default_value,
+    _eval_expression,
+    _get_indexer,
+    _get_mask,
+    _get_valid_ids,
+    _nan_type,
```

Comment:
Would this be useful for the users to be public? Or is there a python side function that does something similar already?

Reply:
we have https://power-grid-model.readthedocs.io/en/stable/api_reference/python-api-reference.html#power_grid_model.typing.ComponentMetaData.nans